internal/admin/dashboard/templates/page-guardrails.html

```
@@ -0,0 +1,232 @@
+                                    <template x-if="field.input !== 'select' && field.input !== 'textarea'">
+                                        <input :id="'guardrail-field-' + field.key" :type="field.input || 'text'" :placeholder="field.placeholder || ''" :value="guardrailFieldValue(field)" :aria-describedby="field.help ? 'guardrail-field-help-' + field.key : null" @input="setGuardrailFieldValue(field, $event.target.value)">
+                                    </template>
+                                </div>
+                            </template>
+                            <template x-if="field.input === 'checkboxes'">
+                                <fieldset class="form-field form-field-wide form-field-fieldset" :aria-describedby="field.help ? 'guardrail-field-help-' + field.key : null">
+                                    <legend class="form-field-legend" x-text="field.label"></legend>
+                                    <div class="workflow-feature-toggles">
+                                        <template x-for="option in field.options" :key="field.key + '-' + option.value">
+                                            <label class="workflow-feature-toggle">
+                                                <input type="checkbox" :checked="guardrailArrayFieldSelected(field, option.value)" @change="toggleGuardrailArrayFieldValue(field, option.value, $event.target.checked)">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)